OpenRA.Game/Traits/Player/Shroud.cs

```diff
@@ -72,6 +72,7 @@ public class Shroud : ISync, INotifyCreated, ITick
 	{
 		public enum SourceType : byte { PassiveVisibility, Shroud, Visibility }
 		public event Action<PPos> OnShroudChanged;
+		public int RevealedCells;
```

Comment:
```suggestion
		public int RevealedCells { get; private set; }
```

For better encapsulation of the data.